DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: epiprocess
 Title: Tools for basic signal processing in epidemiology
-Version: 0.9.0
+Version: 0.10.0
```

Comment:
Increment patch number of a patch?
```suggestion
Version: 0.9.1
```

Reply:
I'm getting mixed up looking at changelog.  Are we building up changes for 0.10 release under 0.9.x?

Reply:
Think of the 0.9 heading like an append log of all the changes in the 0.9.x versions. So we should put the changes from here there.

Reply:
Still confused here.  Going to follow the 0.7.y and 0.8.y pattern of 0.x.y, y > 0, of adding NEWS.md entries for 0.{x+1}, since we can fix up easily later.